services/azure-storage/src/provide_credential/azure_cli.rs

```diff
@@ -96,12 +96,12 @@ impl ProvideCredential for AzureCliCredentialProvider {
 
         // Calculate expiration time
         let expires_on = if let Some(timestamp) = token.expires_on_timestamp {
-            Some(chrono::DateTime::from_timestamp(timestamp, 0).unwrap())
+            Some(jiff::Timestamp::new(timestamp, 0).unwrap())
         } else if let Some(expires_str) = token.expires_on {
             // Parse the string format "2023-10-31 21:59:10.000000"
-            chrono::NaiveDateTime::parse_from_str(&expires_str, "%Y-%m-%d %H:%M:%S%.f")
+            jiff::civil::DateTime::strptime("%Y-%m-%d %H:%M:%S%.f", &expires_str)
```

Comment:
Unless you specifically want to be strict here, you can just use `expires_str.parse::<jiff::civil::DateTime>()` and it will handle `2023-10-31 21:59:10.000000` automatically. (But it will of course allow for other things not allowed by `%Y-%m-%d %H:%M:%S%.f`.)